translations/de/get5.phrases.txt

```diff
@@ -198,26 +322,150 @@
     }
     "TeamSelectSideInfoMessage"
     {
-        "de"            "{1} möchte als {2} starten {3}."
+        "de"            "{1} möchte auf {3} als {2} starten."
     }
     "TeamVetoedMapInfoMessage"
     {
-        "de"            "{1} verbietet {2}."
+        "de"            "{1} legte ein veto gegen {2} ein."
     }
     "CaptainLeftOnVetoInfoMessage"
     {
-        "de"            "Team Captian verließ das Match während der Veto Phase, pausieren..."
+        "de"            "Der Team Captian hat das Spiel während der Vetophase verlassen, daher wird das Veto pausiert."
     }
     "ReadyToResumeVetoInfoMessage"
     {
-        "de"            "Tippe {1} wenn ihr fertig seit mit dem veto fortzufahren."
+        "de"            "Tippe {1} wenn ihr bereit seid mit der Vetophase fortzufahren."
     }
     "MatchConfigLoadedInfoMessage"
     {
-        "de"            "Match Config geladen."
+        "de"            "Spiel Konfiguration geladen."
     }
     "MoveToCoachInfoMessage"
     {
-        "de"            "Das Team ist voll die wurdest als Coach hinzugefügt."
+        "de"            "Das Team ist voll, daher wurdest du als Coach hinzugefügt."
     }
+    "CannotLeaveCoachingTeamIsFull"
+    {
+        "de"            "Du kannst die Coach Position nicht verlassen,da dein Team voll ist."
+    }
+    "CoachingNotEnabled"
+    {
+        "de"            "Coaching ist nicht aktiviert. Um es zu aktivieren, setze {1} auf 1."
+    }
+    "PlayerIsCoachingTeam"
+    {
+        "de"            "{1} ist der Coach von {2}."
+    }
+    "CanOnlyCoachDuringWarmup"
+    {
+        "de"            "Du kannst nur während der Aufwärmphase Coach werden oder die Coach Position verlassen."
+    }
+    "AllCoachSlotsFilledForTeam"
+    {
+        "de"            "Alle Coach Plätze ({1}) für dein Team sind bereits belegt."
+    }
+    "ReadyTag"
+    {
+        "de"            "[Bereit]"
+    }
+    "NotReadyTag"
+    {
+        "de"            "[Nicht bereit]"
```

Comment:
No, because "bereit" is an adjective and they are written in lowercase if not used as a single word to describe the status. But i guess i will simply replicate the english approach and capslock both of them, then there is no right or wrong answer :D